### DISCUSSION TITLE
AKS Automatic Upgrades - Upgrade Groups

### DISCUSSION THREAD

**Discussion by gstsjsts** [Azure/AKS#4003] [Q&A]
I wanted to ask about the status on 'Upgrade Groups' regarding AKS cluster Automatic Upgrades. From the status I can see that it is in 'Public Preview', but I did not find how to enable this feature. Is it a part of the 'Fleet manager'? 

The issue I am facing - I want to enable Automatic Upgrades for 4 different environments, ensuring that the upgrade procedure is sequential, starting from the Development environment up to Production.

**Comment by philwelz** [comment]
Yes, this feature comes with fleet and you can find the docs [here](https://learn.microsoft.com/en-us/azure/kubernetes-fleet/update-orchestration?tabs=azure-portal)
